server/server_test.go

```diff
@@ -120,16 +120,15 @@ func (cli *testServerClient) runTests(t *testing.T, overrider configOverrider, t
 
 // runTestsOnNewDB runs tests using a specified database which will be created before the test and destroyed after the test.
 func (cli *testServerClient) runTestsOnNewDB(t *testing.T, overrider configOverrider, dbName string, tests ...func(dbt *testkit.DBTestKit)) {
-	dsn := cli.getDSN(overrider, func(config *mysql.Config) {
-		config.DBName = ""
-	})
+	dsn := cli.getDSN(overrider)
 	db, err := sql.Open("mysql", dsn)
 	require.NoErrorf(t, err, "Error connecting")
 	defer func() {
 		err := db.Close()
 		require.NoError(t, err)
 	}()
-
+	err = db.Ping()
+	require.NoError(t, err)
```

Comment:
A ping is also included in `waitUntilServerCanConnect`.

Reply:
removed.